### PR TITLE
version 2.0.0

### PR DESCRIPTION
**Pull request by mathieudutour**

### CODE REVIEW

**Comment by jaridmargolin** [comment]
@mathieudutour I know you want to get a release out due to Sketch v50, however I was working through a few additional changes that would be considered "breaking".

Specifically I had toyed with the idea of adjusting the return values of `makeSymbol` in order to support some new workflows (I want to control injecting symbols myself). I may be able to work around breaking changes by introducing new methods rather than repurposing existing methods, but really I would say these issues are to early in development to discuss in detail. What are your thoughts around how often it would acceptable to push "breaking" changes that require major bump?

**Comment by mathieudutour** [comment]
I don't mind publishing breaking changes as long as their usefulness outweigh the cost of updating :)

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Open a new Sketch file, then in a terminal:
 
 ```bash
 git clone https://github.com/airbnb/react-sketchapp.git
-cd react-sketchapp/examples/basic-setup && npm install
+cd react-sketchapp/examples/basic-setup && npm installLobby
```
Comment (jaridmargolin) [inline README.md]: Is this right?

**Comment by jaridmargolin** [comment]
I don't mind giving this the green light... but wondering if you are waiting for feedback from airbnb folks before releasing a new major bump.

**Pull request merged**